core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4690,6 +4721,30 @@ paths:
                 $ref: '#/components/schemas/ErrorReport'
         500:
           $ref: '#/components/responses/RawlsInternalError'
+  /api/googleProjects/{googleProjectId}:
+    get:
+      tags:
+        - googleProjects
+      summary: get google project by googleProjectId
+      description: Get the Google project if known and the user has access
+      operationId: getGoogleProjectById
+      parameters:
+        - $ref: '#/components/parameters/googleProjectIdPathParam'
+      responses:
+        200:
+          description: Successful Request
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/RegisterGoogleProjectRequest'
```

Comment:
for both of these APIs, I think we should also return at least the `message` on the project (if it exists). We may also want to return the billing account - not sure about that one; returning the billing project id might be enough.